build/csv/ceph/rook-ceph-operator.clusterserviceversion.yaml

```diff
@@ -1863,12 +1863,22 @@ metadata:
   name: rook-ceph-operator.v4.15.0
   namespace: placeholder
   relatedImages:
```

Comment:
I believe this is wrong section for relatedImages, other operators have it in `spec`.

Reply:
updated